package.json

```diff
@@ -11,7 +11,7 @@
     "@babel/runtime": "7.14.0",
     "@fortawesome/fontawesome-free": "5.15.3",
     "@popperjs/core": "2.9.2",
```

Comment:
Also tried to remove that, but that gave me an error.. do you have an idea why?

Reply:
no?! Bootstrap has it's own popper dependency so we shouldn't need this?! will have a little poke

Reply:
ok added popper to a4 once merged we can update this hash and rm it from here